scripts/deploy/check-monitors.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Check monitors status
+ * Usage:
+ * node check-monitors.js us1,eu1,...
+ */
+const { printLog, runMain, fetch } = require('../lib/execution-utils')
+const { getTelemetryOrgApiKey, getTelemetryOrgApplicationKey } = require('../lib/secrets')
+const { siteByDatacenter } = require('../lib/datadog-sites')
+
+const monitorIdsByDatacenter = {
+  us1: [72055549, 68975047, 110519972],
+  eu1: [5855803, 5663834, 9896387],
+  us3: [164368, 160677, 329066],
+  us5: [22388, 20646, 96049],
+  ap1: [858, 859, 2757030],
+}
+
+const datacenters = process.argv[2].split(',')
+
+runMain(async () => {
+  for (const datacenter of datacenters) {
+    if (!monitorIdsByDatacenter[datacenter]) {
+      printLog(`No monitors configured for datacenter ${datacenter}`)
+      continue
+    }
+    const monitorIds = monitorIdsByDatacenter[datacenter]
+    const site = siteByDatacenter[datacenter]
+    const monitorStatuses = await Promise.all(monitorIds.map((monitorId) => fetchMonitorStatus(site, monitorId)))
+    for (const monitorStatus of monitorStatuses) {
+      printLog(`${monitorStatus.overall_state} - ${monitorStatus.name}`)
+      if (monitorStatus.overall_state !== 'OK') {
+        throw new Error(
+          `Monitor ${monitorStatus.name} is in state ${monitorStatus.overall_state}, see ${computeMonitorLink(site, monitorStatus.id)}`
+        )
+      }
+    }
+  }
+})
+
+async function fetchMonitorStatus(site, monitorId) {
+  const response = await fetch(`https://api.${site}/api/v1/monitor/${monitorId}`, {
+    method: 'GET',
+    headers: {
+      Accept: 'application/json',
+      'DD-API-KEY': getTelemetryOrgApiKey(site),
+      'DD-APPLICATION-KEY': getTelemetryOrgApplicationKey(site),
+    },
+  })
+  return JSON.parse(response)
```

Comment:
**💭 thought:** ‏this function called `fetch` is quite confusing (for example it's not possible to use `JSON.parse(response)` like this with the native `fetch` function), I'd like to move away from this name clash one day!

Reply:
Actually, do we still to use this fetch wrapper ?
https://github.com/DataDog/browser-sdk/blob/84cd328da65f6957208bc4b2dd2fec21e39de870/scripts/lib/execution-utils.js#L3

Reply:
We definitly can remove `node-fetch`. But we need to see what to do with the wrapper: https://github.com/DataDog/browser-sdk/blob/84cd328da65f6957208bc4b2dd2fec21e39de870/scripts/lib/execution-utils.js#L54-L61
Exporting it as `fetch` is confusing.

Reply:
I'll look into in a following PR